hack/tests/subcommand-generate-csv.sh

```diff
@@ -41,8 +41,8 @@ function check_crd_files() {
 }
 
 function generate_csv() {
-  echo "operator-sdk generate csv --operator-name $OPERATOR_NAME $@"
-  operator-sdk generate csv --operator-name $OPERATOR_NAME $@
+  echo "operator-sdk generate csv --operator-name $OPERATOR_NAME $@ --interactive=false"
```

Comment:
Nit: best practice is to put `$@` at the end of the string you're expanding:
```suggestion
  echo "operator-sdk generate csv --operator-name $OPERATOR_NAME --interactive=false $@"
```

Ditto below.